Library/Formula/moose.rb

```diff
@@ -0,0 +1,60 @@
+class Moose < Formula
+  desc "MOOSE is a Multiscale Object Oriented Simulation Environment for simulating multiscale neural systems."
```

Comment:
Please don't start formula descriptions with the formula name - That happens automatically.